Examples/Makefile.in

```diff
@@ -1819,14 +1819,24 @@ go: $(SRCDIR_SRCS)
 go_cpp: $(SRCDIR_SRCS)
 	$(SWIG) -go -c++ $(GOOPT) $(GOSWIGARG) $(SWIGOPT) -o $(ICXXSRCS) $(INTERFACEPATH)
 	if $(GO12) || $(GO13) || $(GOGCC); then \
-	  $(CXX) -g -c $(CPPFLAGS) $(CXXFLAGS) $(SRCDIR_SRCS) $(SRCDIR_CXXSRCS) $(ICXXSRCS) $(INCLUDES); \
+	  if test -n "$(SRCDIR_CXXSRCS)$(SRCDIR_SRCS)"; then \
+	    $(CXX) -g -c $(CPPFLAGS) $(CXXFLAGS) $(SRCDIR_CXXSRCS) $(SRCDIR_SRCS) $(INCLUDES); \
+	  fi; \
+	  ${foreach f,$(ICXXSRCS), \
+	    $(CXX) -g -c $(CPPFLAGS) $(CXXFLAGS) -o ${addsuffix .o,${basename $f}} $f $(INCLUDES); \
```

Comment:
The other function calls in this Makefile consistently use $(), not ${}.  Seems that this part may as well be consistent.